hedera-node/hedera-smart-contract-service-impl/src/test/java/com/hedera/node/app/service/contract/impl/test/exec/ContextTransactionProcessorTest.java

```diff
@@ -236,7 +254,44 @@ void stillChargesHapiFeesOnAbort() {
     }
 
     @Test
-    void stillChargesHapiFeesOnHevmException() {
+    void chargeHapiFeeOnFailedEthTransaction() {
```

Comment:
Tests here don't seem to test with the new flag _both_ disabled and enabled.

Reply:
extended existing tests to verify this depending on the config